ui/src/button.test.tsx

```diff
@@ -254,5 +255,93 @@ describe('Button.tsx', () => {
       expect(getByTestId(name)).toHaveClass('ms-Link')
       expect(container.querySelector('i[data-icon-name="ChevronDown"]') as HTMLLIElement).not.toBeInTheDocument()
     })
+
+    it('Does not set args and calls sync on click when command has download link specified', () => {
+      const
+        btnCommandDownloadProps: Buttons = {
+          items: [{
+            button: {
+              name,
+              label: name,
+              commands: [
+                { name: 'command1', label: 'Command 1' },
+                { name: 'command2', label: 'Command 2', path, download: true },
+              ]
+            }
+          }]
+        },
+        { container, getByText } = render(<XButtons model={btnCommandDownloadProps} />),
+        contextMenuButton = container.querySelector('i[data-icon-name="ChevronDown"]') as HTMLLIElement
+
+      expect(wave.args['command1']).toBe(false)
+      fireEvent.click(contextMenuButton)
+      fireEvent.click(getByText('Command 1'))
+      expect(wave.args['command1']).toBe(true)
+
+      expect(pushMock).toHaveBeenCalled()
+      expect(pushMock).toHaveBeenCalledTimes(1)
+
+      expect(wave.args['command2']).toBe(false)
+      fireEvent.click(contextMenuButton)
+      fireEvent.click(getByText('Command 2'))
+      expect(wave.args['command2']).toBe(false)
+
+      expect(pushMock).toHaveBeenCalledTimes(1)
+    })
+
+    it('Ignores items when command has download link specified', () => {
```

Comment:
What does this test do? I couldn't figure it out.

Reply:
It tests [this line of code ](https://github.com/h2oai/wave/blob/6712a4746994f69b3cae5f38052c037defa8a686/ui/src/card_menu.tsx#L84) to verify that command having `path` specified cannot have context menu with another sub-commands.

At least for me it does not make sense for command to be clickable in case of having context menu.

Reply:
Are subcommands invoked by a click? I thought they appear on hover.

Reply:
Got it, makes sense. Ty.